uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -40,6 +40,7 @@ define([
       this.requestDayTypes = [];
       this.selectedAbsenceType = {};
       this.period = {};
+      this.postContactSelection = false, //flag to track if user is selected for enabling UI
```

Comment:
If we really can't avoid this additional flag, then at least let's put it together with the other `loading.{x}` flags and give it a more fitting name, something like `this.loading.postContactSelectionData`. I consider this temporary, there must be some improvements to the code re how we deal with loading statuses